src/coreComponents/python/modules/pygeosx_tools_package/pygeosx_tools/file_io.py

```diff
@@ -0,0 +1,53 @@
+
+import os
+import numpy as np
+
+
+def save_tables(grid, properties, table_root='./tables', axes=['x', 'y', 'z', 't']):
+    """
+    Saves a set of tables in GEOSX format
+    Notes: The shape of these arrays should match the length of each axis in the specified order
+           The output directory will be created if it does not exist yet
+
+    Args:
+        grid (list): A list of numpy ndarrays defining the table axes
+        properties (dict): A dict of numpy ndarrays defning the table values
+        table_root (str): The root path for the output directory
+        axes (list): A list of names for each potential axis (default = ['x', 'y', 'z', 't'])
+    """
+    os.makedirs(table_root, exist_ok=True)
+    for ii, d in zip(range(len(grid)), axes):
+        np.savetxt('%s/%s.csv' % (table_root, d),
+                   grid[ii],
+                   fmt='%1.5f',
+                   delimiter=',')
```

Comment:
Why not simply
```suggestion
    for g, a in zip(grid, axes):
        np.savetxt('%s/%s.csv' % (table_root, a),
                   g,
                   fmt='%1.5f',
                   delimiter=',')
```
? Is it equivalent?

Reply:
Agreed, I've cleaned up the iterators for this and the following block